### DISCUSSION TITLE
Unable to connect GCP Cloud MySQL Instance

### DISCUSSION THREAD

**Discussion by aliuygur** [directus/directus#12946] [Archive]
Directus unable to connect GCP Cloud MySQL instance since that is using socket instead of tcp host.

Here is solution of this problem.
https://stackoverflow.com/a/52502213

### To Reproduce

Create GCP Cloud Mysql 8 Instance
Deploy Directus with GCP Cloud Run and Connect MySQL instance

### Errors Shown

Error: getaddrinfo EAI_AGAIN /cloudsql/jetbuk:europe-west1:jetbukpq
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:71:26)

<details>
<summary>More info</summary>

### What version of Directus are you using?

9.9.0

### What version of Node.js are you using?

16

### What database are you using?

MySQL 8

### What browser are you using?

Chrome

### How are you deploying Directus?

GCP Cloud Run with Cloud MySQL

</details>

**Comment by azrikahar** [comment]
With #7420 as the reference here, you should be able to use `DB_SOCKET_PATH` to achieve this 👍 which will translate to `socketPath` config of the `mysql` driver: https://www.npmjs.com/package/mysql#connection-options

With that said, I do believe this particular constraint: https://github.com/directus/directus/discussions/7420#discussioncomment-1197411 is still not lifted, so you may still need satisfy the required check by passing `DB_HOST` and `DB_PORT` anyway, even if those two will get ignored when `DB_SOCKET_PATH` is used as mentioned in the `mysql` docs.
